tests/README.md

```diff
@@ -11,7 +13,7 @@ This means that we extract and test the exact same commands that are published i
 These tests use the framework defined in the `istioio` package, which is a thin wrapper
 around the [Istio test framework](https://github.com/istio/istio/wiki/Istio-Test-Framework).
 
-Run the following command to see the current test coverage, including the list of documents
+[UPDATE NEEDED] Run the following command to see the current test coverage, including the list of documents
```

Comment:
Should we add TODOs and reference tracking bugs?

Reply:
What steps does "reference tracking bugs" translate to? I'm not sure if there's an inventory of TODOs to keep them.

Reply:
Ah right ... Usually we do something like `TODO(https://github.com/istio/istio/issues/<number>`. It just helps readers know that this work is scheduled and gives them a way to track the progress.